### ISSUE TITLE
trouble mounting config file

### ISSUE THREAD

**Issue by jill88** [Bleala/Vdirsyncer-DOCKERIZED#34]
I think (I'm not sure) I am having trouble mounting the config file because it doesn't have an extension.  I feel docker compose keeps treating it as a folder, but it's just a file.  Am I misunderstanding something?  I have tried every which way, thanks.  Here is my compose file.  Whenever I start the container, it just exits saying it can't create the log file, but I think it is not reading the config file at all.

```
networks:
  default:
    driver: bridge
volumes:
  vdirsyncer:
    name: vdirsyncer
    driver: local
services:
  # Vdirsyncer - sync calendars and addressbooks between servers and the local filesystem. DOCKERIZED!
  # https://hub.docker.com/r/bleala/vdirsyncer
  app:
    image: bleala/vdirsyncer:latest
    container_name: vdirsyncer
    restart: unless-stopped
    networks:
      - default
    environment:
      - TZ=America/Los_Angeles # set your timezone, for correct container and log time, default to Europe/Vienna
      - AUTODISCOVER= # set to true for automatic discover, default to false
      - AUTOSYNC= # set to true for automatic sync, default to false
      - AUTOUPDATE= # set to true for automatic Vdirsyncer and dependencies updates on container startup, default to false
      - CRON_TIME= # adjust autosync /-discover time, default to 15 minutes - */15 * * * * 
      - POST_SYNC_SCRIPT_FILE= # optional,  set to script path to automatically run your custom script after the cronjob `vdirsyncer` command(s), default to nothing
      - LOG= # optional, default to /vdirsyncer/vdirsyncer.log
      - LOG_LEVEL= # optional, default is normal output from supercronic
    volumes:
      - vdirsyncer:/vdirsyncer # Docker Volume
      - ./config:/vdirsyncer/config # Vdirsyncer Config
      # Optional
      # - /path/to/custom_script.sh:/vdirsyncer/custom_script.sh  # Custom Script

```

**Comment by jill88** [comment]
OK i got it working because i had to put in an absolute path for the log file.  I didn't notice that my vdirsyncer directory was a different one than the default.
Now, my trouble is getting google contacts to work,  Boy this one is tough.  I am trying to use that 9-step workaround mentioned in the docs, but having a hard time.  I am using an "external" oath, because i don't have internal available, but also a bunch of other settings there like domains I don't quite understand.

**Comment by jill88** [comment]
OK I got a little bit more working.  I had to use the 127.0.0.1 uri redirect like I saw in the other thread.  I got the code from the url address and everything.  But I can't see where I paste this code in anywhere..  What do I do now?  The console where I am running the ssh commands is just hanging there.  nothing happens if I paste the code there, so I am at a loss.

**Comment by Bleala** [comment]
Hi @jill88 

You have to do it like this: https://github.com/pimutils/vdirsyncer/issues/1063#issuecomment-1910758500
Open a second terminal inside the container and execute the curl command.
This should work.

Greetings

**Comment by jill88** [comment]
I keep trying that, but I can't get it to continue.  I curl the url using the docker exec command, but it doesn't do anything.  It returns two lines of some numbers, but the prompt itself doesn't continue.

**Comment by jill88** [comment]
OK I got it all working now.  What a pain google is.  so i was able to run the command from the container.  I had to get into the sh prompt first before running the curl command.  But then the other thing with google is that the addressbook name has to be "default" .  Also, during the sync,, i don't know if it's google or the sync, but the high quality photos degrade a lot after syncing from radicale to google.
